src/components/Common/UI/Calendar/CalendarTypes.ts

```diff
@@ -0,0 +1,12 @@
+export type CalendarProps = {
+  highlightDates?: Array<{
+    date: Date
```

Comment:
Updated to use `Date` on these instead of `string` to be more clear with consumers using this API what data they are working with. This also makes this consistent with `DatePicker` which uses `Date`